File: Data Structures & Algorithms/baseball-game/submission-0.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        sum = 0
        ops = []
        for i in operations:
            if i=="D":
                double = 2*ops[-1]
                ops.append(double)
                sum+=double
            elif i=="+":
                add=ops[-1]+ops[-2]
                ops.append(add)
                sum+=int(add)
            elif i=="C":
                remove=ops.pop()
                sum-=remove
            else:
                score=int(i)
                ops.append(score)
                sum+=score
        return sum

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n=nums.size(),i,j,sum=0;
        for(i=0;i<n;i++){
            for(j=i+1;j<n;j++){
                sum=nums[i]+nums[j];
                if(sum==target){
                    return {i, j};
                }
            }
        }
        return {};
    }
};
